package.json

```diff
@@ -4,6 +4,7 @@
   "description": "This is a Workshop about Cypress",
   "main": "index.js",
   "scripts": {
+    "cypress": "cypress open",
```

Comment:
why you added this, the test:open will also make the same action

Reply:
That line should be deleted, that was something I was testing from the "Cypress Ramp-Up" course.   I will delete it next PR.

Reply:
@holgiosalo s and definitely it is not the same, because test:open has included the code check instruction **npm run lint** which takes more time to execute.